deploy/helm/kubernetes/values.yaml

```diff
@@ -3,3 +3,60 @@ apiServer:
   certSANs:
     dnsNames: []
     ipAddresses: []
+
+imagePullPolicy: IfNotPresent
+
+etcd:
+  image: k8s.gcr.io/etcd:3.4.3-0
+  podManagementPolicy: Parallel
```

Comment:
I have a bit fear about this, are you sure that we need prefer `Parallel` instead of `OrderedReady`, why?

Reply:
Your choice. [etcd-statefulset.yaml](https://github.com/kvaps/kubernetes-in-kubernetes/blob/bb1f5cc5ce7a4683ec8bf2ce119105d63e3ad489/deploy/helm/kubernetes/templates/etcd-statefulset.yaml#L28).

Replaced.

Reply:
Oh, I see, you just took this from my manifest, I don't know where it comes from, probably some copy&paste bug, let's omit this.

Reply:
removed